src/components/Comments/Comment.js

```diff
@@ -199,7 +279,7 @@ const Comment = ({
           </ButtonHightlight>
         </TouchableOpacity>
 
-        <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
+        {totalVote != 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition check before rendering the total vote count is a good practice to avoid unnecessary rendering when there are no votes.
```diff
+ {totalVote != 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->